manifests/m/Mattermost/MattermostDesktop/4.7.0/Mattermost.MattermostDesktop.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Mattermost.MattermostDesktop
+PackageVersion: 4.7.0
+PackageLocale: en-US
+Publisher: Mattermost
```

```suggestion
Publisher: Mattermost, Inc.
```
Quick change to match ARP